src/TestStack.White/Mappings/ControlDictionaryItems.cs

```diff
@@ -18,6 +18,9 @@ public virtual void AddWPFPrimary(Type testControlType, ControlType controlType)
         }
 
         public virtual void AddXamlPrimary(Type testControlType, ControlType controlType)
+        {
+            Add(ControlDictionaryItem.XamlPrimary(testControlType, controlType));
+        }
```

Comment:
This was a quick guess at what text had gone missing. I'm not at all sure though.